jujugui/static/gui/src/app/components/local-inspector/local-inspector.js

```diff
@@ -175,7 +175,8 @@ class LocalInspector extends React.Component {
       <div className="inspector-view local-inspector">
         <InspectorHeader
           backCallback={this._close.bind(this)}
-          title="Local charm" />
+          title="Local charm"
```

Comment:
Were these changes to local-inspector and test-local-inspector supposed to have been added? They don't seem to change anything